### PR TITLE
chore(links): update links

### PR DESCRIPTION
**Pull request by Jolg42**
- some links changed (and were being redirected), so I added the current version
- we switched to https://pris.ly/prisma-prisma-bug-report for links for bug reports see https://github.com/prisma/docs/pull/5449

### CODE REVIEW

File: packages/cli/README.md [hunk 1]
```diff
@@ -34,16 +34,16 @@ Prisma is a **next-generation ORM** that consists of these tools:
...
 - [**Prisma Studio**](https://github.com/prisma/studio): GUI to view and edit data in your database
 
-Prisma Client can be used in _any_ Node.js or TypeScript backend application (including serverless applications and microservices). This can be a [REST API](https://www.prisma.io/docs/understand-prisma/prisma-in-your-stack/rest), a [GraphQL API](https://www.prisma.io/docs/understand-prisma/prisma-in-your-stack/graphql) a gRPC API, or anything else that needs a database.
+Prisma Client can be used in _any_ Node.js or TypeScript backend application (including serverless applications and microservices). This can be a [REST API](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/rest), a [GraphQL API](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/graphql) a gRPC API, or anything else that needs a database.
 
 ## Getting started
 
-The fastest way to get started with Prisma is by following the [**Quickstart (5 min)**](https://www.prisma.io/docs/getting-started/quickstart-typescript).
+The fastest way to get started with Prisma is by following the [**Quickstart (5 min)**](https://www.prisma.io/docs/getting-started/quickstart).
```
Comment (ruheni) [inline packages/cli/README.md:41]:
```suggestion
The fastest way to get started with Prisma is by following the [**Quickstart (5 min)**](https://pris.ly/quickstart).
```

File: README.md [hunk 2]
```diff
@@ -36,16 +36,16 @@ Prisma is a **next-generation ORM** that consists of these tools:
...
 - [**Prisma Studio**](https://github.com/prisma/studio): GUI to view and edit data in your database
 
-Prisma Client can be used in _any_ Node.js or TypeScript backend application (including serverless applications and microservices). This can be a [REST API](https://www.prisma.io/docs/understand-prisma/prisma-in-your-stack/rest), a [GraphQL API](https://www.prisma.io/docs/understand-prisma/prisma-in-your-stack/graphql), a gRPC API, or anything else that needs a database.
+Prisma Client can be used in _any_ Node.js or TypeScript backend application (including serverless applications and microservices). This can be a [REST API](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/rest), a [GraphQL API](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/graphql), a gRPC API, or anything else that needs a database.
 
 ## Getting started
 
-The fastest way to get started with Prisma is by following the [**Quickstart (5 min)**](https://www.prisma.io/docs/getting-started/quickstart-typescript).
+The fastest way to get started with Prisma is by following the [**Quickstart (5 min)**](https://www.prisma.io/docs/getting-started/quickstart).
```
Comment (ruheni) [inline README.md:43]:
```suggestion
The fastest way to get started with Prisma is by following the [**Quickstart (5 min)**](https://pris.ly/quickstart).
```

Comment (ruheni) [reply README.md:43]: Found a pris.ly link on dub. I think we could use it for tracking..

**Review by ruheni — COMMENTED** [review]
LGTM. I've left a few suggestions

**Review by janpio — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Jolg42) [reply README.md:43]: and it's easier to change it!

**Pull request merged**